### DISCUSSION TITLE
求助：在mpvnet_cm中如何开启rife_nv

### DISCUSSION THREAD

**Discussion by mmnmuo** [hooke007/mpv_PlayKit#487] [libmpv]
在mpvnet_cm中启用英伟达显卡专用的rife补帧该怎么做？抑或是要调整哪些文件？以及这些模型的官网在哪里？我怎么翻都翻不到

我像个中世纪炼金术士一样的，下载了mpv_lazy以及其扩展包vsmaga，也下载了mpvnet_cm，把里面的文件轮流倒，然后报一堆错🤡

**Comment by zhongfly** [comment]
所以，为啥不先看文档，先学会怎么在mpv里用vapoursynth，再用vapoursynth调用rife模型.
这里都有相应的文档

0. https://hooke007.github.io/unofficial/mpv_start.html#vapoursynth
1. https://github.com/hooke007/MPV_lazy/wiki/3_K7sfunc#1-%E4%BD%BF%E7%94%A8%E5%BC%95%E5%AF%BC
2. https://github.com/hooke007/MPV_lazy/wiki/3_K7sfunc#rife_nv

**Reply by mmnmuo** [reply]
看过了不止一遍。。。可能是融会贯通的能力不够强吧，我再去瞪一遍

**Reply by zhongfly** [reply]
> 看过了不止一遍。。。可能是融会贯通的能力不够强吧，我再去瞪一遍

你安装的vapoursynth是什么版本？R66及以后的安装有变化的

**Reply by zhongfly** [reply]
不同版本vapoursynth需要的python版本也是不完全相同

另外 建议按照这个版本搭配，省事一些又新一点的话 python3.11 + VapourSynth R62-R65
https://gist.github.com/hooke007/e502688c60ef1c5f9ef507cf4db25b34

**Reply by mmnmuo** [reply]
我直接下载了mpvnet_cm，用的应该就是里面的版本吧。。

**Reply by zhongfly** [reply]
那就不用安装vapoursynth了，已经自带了.
只是不知道k7sfunc在旧版环境会不会有问题。

0. 把 https://github.com/hooke007/MPV_lazy/blob/main/k7sfunc.py 放到mpvnet.exe同文件夹
1. 安装rife_nv需要的其他依赖： https://github.com/hooke007/MPV_lazy/wiki/3_K7sfunc#rife_nv

那我只能建议别用mpvnet_cm了，废弃很久了

**Reply by mmnmuo** [reply]
我从lazy包里把所要求的plugins，vpy文件都拖了过来，input也加了一行，报了一个没找到model的错，然后我又从lazy包把所有model拖了过来，现在是既不报错，切换到我设置的预设一也没反应。
能不能帮我猜猜是什么问题，我漏掉了什么吗
![Uploading 微信图片_20240722223947.png…]()

**Reply by zhongfly** [reply]
哦，看起来还需要用pip安装numpy、onnx，你还得学习一下 https://gist.github.com/hooke007/e502688c60ef1c5f9ef507cf4db25b34#21-%E5%AE%89%E8%A3%85-pip-%E5%8F%AF%E9%80%89

**Reply by zhongfly** [reply]
我不用vs的，你自己琢磨吧。别老想着直接复制就搞定，事实证明了不行。老老实实一步一步做吧，直接复制没用的。mpv_lazy的vs还有python版本估计和cm自带的都不一样

**Reply by mmnmuo** [reply]
谢谢老哥提供的链接，经过漫长的努力，总算能用了

**Reply by AyTsaoIce** [reply]
> 谢谢老哥提供的链接，经过漫长的努力，总算能用了

mmnmuo能问问是如何实现的吗，或者步骤是什么，我卡在
>print(core.list_functions())
Traceback (most recent call last):
File "", line 1, in
File "src\cython\vapoursynth.pyx", line 2434, in vapoursynth._CoreProxy.getattr
File "src\cython\vapoursynth.pyx", line 2308, in vapoursynth.Core.getattr
AttributeError: No attribute with the name list_functions exists. Did you mistype a plugin namespace?

就无法推进了

**Reply by hooke007** [reply]
cm带的vs太老了，删掉

**Reply by AyTsaoIce** [reply]
> cm带的vs太老了，删掉

谢谢大佬指路(*/ω＼*)我去试试
——————
[edited] 感谢，删除了旧版本的vs之后能够成功使用了，麻烦了
